sdk/core/core-http/src/serviceClient.ts

```diff
@@ -332,7 +333,8 @@ export class ServiceClient {
           urlParameterValue = operationSpec.serializer.serialize(
             urlParameter.mapper,
             urlParameterValue,
-            getPathStringFromParameter(urlParameter)
+            getPathStringFromParameter(urlParameter),
+            { xmlCharKey: operationArguments.options?.parsingOptions?.xmlCharKey }
```

Comment:
can we create this object once earlier in the method and pass it by reference in each place?